google-dracut-config.spec

```diff
@@ -39,34 +39,36 @@ BuildArch:      noarch
 %install
 
 install -D -m 644 etc/dracut.conf.d/07-ext4.conf \
-    %{buildroot}%{_sysconfdir}/dracut.conf.d/07-ext4.conf
+    %{buildroot}%{_libexecdir}/dracut.conf.d/07-ext4.conf
 install -D -m 644 etc/dracut.conf.d/07-net.conf \
-    %{buildroot}%{_sysconfdir}/dracut.conf.d/07-net.conf
+    %{buildroot}%{_libexecdir}/dracut.conf.d/07-net.conf
 install -D -m 644 etc/dracut.conf.d/07-nvme.conf \
-    %{buildroot}%{_sysconfdir}/dracut.conf.d/07-nvme.conf
+    %{buildroot}%{_libexecdir}/dracut.conf.d/07-nvme.conf
 install -D -m 644 etc/dracut.conf.d/07-virtio.conf \
-    %{buildroot}%{_sysconfdir}/dracut.conf.d/07-virtio.conf
+    %{buildroot}%{_libexecdir}/dracut.conf.d/07-virtio.conf
 install -D -m 644 etc/dracut.conf.d/07-xfs.conf \
-    %{buildroot}%{_sysconfdir}/dracut.conf.d/07-xfs.conf
+    %{buildroot}%{_libexecdir}/dracut.conf.d/07-xfs.conf
 install -D -m 644 etc/dracut.conf.d/11-resume.conf \
-    %{buildroot}%{_sysconfdir}/dracut.conf.d/11-resume.conf
+    %{buildroot}%{_libexecdir}/dracut.conf.d/11-resume.conf
 
 %check
 
 %post
 has_idpf=$(find /lib/modules -name 'idpf*.ko*')
 if [ -z "$has_idpf" ]; then
-    sed -i s/idpf// %{_sysconfdir}/dracut.conf.d/07-net.conf
+    sed -i s/idpf// %{_libexecdir}/dracut.conf.d/07-net.conf
 fi
 
 %files
 %doc README.md
 %license LICENSE
-%config %{_sysconfdir}/dracut.conf.d/07-ext4.conf
-%config %{_sysconfdir}/dracut.conf.d/07-net.conf
-%config %{_sysconfdir}/dracut.conf.d/07-nvme.conf
-%config %{_sysconfdir}/dracut.conf.d/07-virtio.conf
-%config %{_sysconfdir}/dracut.conf.d/07-xfs.conf
-%config %{_sysconfdir}/dracut.conf.d/11-resume.conf
+%dir %{_libexecdir}dracut/
+%dir %{_libexecdir}dracut/dracut.conf.d/
+%{_libexecdir}dracut/dracut.conf.d/07-ext4.conf
+%{_libexecdir}dracut/dracut.conf.d/07-net.conf
+%{_libexecdir}dracut/dracut.conf.d/07-nvme.conf
+%{_libexecdir}dracut/dracut.conf.d/07-virtio.conf
+%{_libexecdir}dracut/dracut.conf.d/07-xfs.conf
+%{_libexecdir}dracut/dracut.conf.d/11-resume.conf
```

Comment:
As mentioned in https://github.com/SUSE-Enceladus/microsoft-dracut-config/pull/2, `%_libexecdir`  expands to `/usr/libexec` on some distros and is intended for executables used indirectly. This should be `%_libdir` or `/usr/lib`, at least until the dracut package gets changed to use `/usr/etc`.

Reply:
Fixed. Using full path instead